examples/Joints/Ingen/pinjoint.py

```diff
@@ -0,0 +1,16 @@
+from sympy import Symbol
```

Comment:
This is the same example as simple_pinjoint but using Ingen's API. It gives the same equations of motion.